src/modules/csm/types.py

```diff
@@ -10,8 +10,43 @@
 logger = logging.getLogger(__name__)
 
 
+class StrikesList(Sequence[int]):
+    """Deque-like structure to store strikes"""
+
+    sentinel: int
```

Comment:
Can we just use hardcoded `0` in `resize` ? To simplify the thing

Reply:
I end up realizing I want to keep the value, but as a class-level constant to avoid using magic numbers.